app/src/main/java/pm/gnosis/android/app/authenticator/GnosisAuthenticatorApplication.kt

```diff
@@ -19,6 +20,7 @@ class GnosisAuthenticatorApplication : Application() {
             Timber.plant(DebugTree())
         }
 
+        LinuxSecureRandom()
```

Comment:
We are crashing the app if that is not possible. Any idea.how to do that better?

Reply:
Yes. Surrounding with try catch should work because when we try to get the `SecureRandom` it will use the default implementation